webui/src/Main.elm

```diff
@@ -335,23 +313,6 @@ notFoundBody model =
     div [] [ text "This is the notFound view" ]
 
 
-viewHeader : Model -> Html Msg
-viewHeader model =
```

Comment:
This set of deletions was surprising, are these really not called?

Reply:
This function is not called at all in the codebase.

Reply:
thanks for catching these! great to have the dead code removed.